test/renate/updaters/test_model_updater.py

```diff
@@ -32,6 +33,39 @@ def test_simple_model_updater(tmpdir, provide_folder):
     assert not torch.allclose(y_hat_before_train, y_hat_after_train)
 
 
+def test_deterministic_updater():
+    # The behavior is always deterministic on CPU but it can become non-deterministic on GPU
```

Comment:
given that we use no GPU, this test is not checking what the name implies. I think we can keep a test like this but it would test whether seeding works (maybe we have this one already somewhere).

it would be sufficient to test whether `deterministic_trainer` is forwarded to `Trainer` since we can assume that Lightning is working. I'm not sure we have a good way to test that.

Reply:
I agree with your point, it's written also in the comment on lines 37 and 38.
I was more thinking that this can run on GPU with the integration tests.

Reply:
integration tests will not run the unit tests. for completeness, one would also need to check the False case